src/store/modules/exampleModule.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export default {
+
+    state: {
+        isLoggedIn: false,
+        errorMsg: "login failed",
+        // axiosInstance: null,
+        tokenAuth: false
+    },
+    mutations: {
+        "login": (state) => {
+            state.isLoggedIn = true
+        },
+        "Token/login": (state) => {
+            state.tokenAuth = true
+        }
+    },
+    getters: {
+
+        hasToken(state) { return state.tokenAuth }
+
+    },
+    actions: {
+        login: async (context, payload) => {
+
+            await axios.post('https://claplablmsapi.clap.co.il/auth/login/1',
+                {
+                    email: payload.email,
+                    password: payload.password
+                }
+            ).then((response) => {
+                localStorage.setItem('access_token', response.data.token)
+                context.commit("login");
+                context.commit("Token/login");
+
+            });
+        },
+        "Token/login": async (context) => {
+            await axios
+                .get("https://claplablmsapi.clap.co.il/auth/checkLogin", {
+                    headers: {
+                        Authorization: `Bearer ${localStorage.getItem("access_token")}`,
+                    },
```

Comment:
you call is using "async/await" AND "then"
u need to choose, both of those methods handle promises.
use only the async await (its the newer feature) remove the ".then"